wolfcrypt/src/aes.c

```diff
@@ -10064,7 +10184,7 @@ int  wc_AesCcmDecrypt(Aes* aes, byte* out, const byte* in, word32 inSz,
 
 #ifdef WOLF_CRYPTO_CB
     if (aes->devId != INVALID_DEVID) {
-        int ret = wc_CryptoCb_AesCcmDecrypt(aes, out, in, inSz, nonce, nonceSz,
+        ret = wc_CryptoCb_AesCcmDecrypt(aes, out, in, inSz, nonce, nonceSz,
```

Comment:
Consider setting `ret = 0` after the `if (ret != CRYPTOCB_UNAVAILABLE)` below.

Reply:
OK what I've done in the end to clarify the code, is use `int crypto_cb_ret` locally for each of the `wc_CryptoCb_*()` calls.  This gives us the best of all worlds I think.